code/TemplateStudioForAvalonia/Templates/_comp/MT/Project.FSharp.Android/Param_ProjectName.Android/Activities_postaction.fs

```diff
@@ -1,37 +1,26 @@
 namespace Param_RootNamespace.Android
+
 open Android.App
-open Android.Content
 open Android.Content.PM
 open Avalonia
+open Avalonia.Android
 //{--{
 open Avalonia.ReactiveUI
 //}--}
-type Application = Android.App.Application
-
-open Avalonia.Android
 open Param_RootNamespace
 
 [<Activity(
     Label = "Param_RootNamespace.Android",
     Theme = "@style/MyTheme.NoActionBar",
     Icon = "@drawable/icon",
-    LaunchMode = LaunchMode.SingleTop,
-    ConfigurationChanges = (ConfigChanges.Orientation ||| ConfigChanges.ScreenSize))>]
+    MainLauncher = true,
+    ConfigurationChanges = (ConfigChanges.Orientation ||| ConfigChanges.ScreenSize ||| ConfigChanges.UiMode))>]
 type MainActivity() =
-    inherit AvaloniaMainActivity()
-
-
-[<Activity(Theme = "@style/MyTheme.Splash", MainLauncher = true, NoHistory = true)>]
-type SplashActivity() =
-    inherit  AvaloniaSplashActivity<App>()
+    inherit AvaloniaMainActivity<App>()
 
     override _.CustomizeAppBuilder(builder) =
         base.CustomizeAppBuilder(builder)
             .WithInterFont()
             //{--{
```

Comment:
Somehow the `.UseReactiveUI()` call does not get removed when community toolkit is selected only for F#

Reply:
No idea why it happens so i have decided to replace the whole file with the correct one